stdlib/2.7/collections.pyi

```diff
@@ -7,8 +7,9 @@
 # NOTE: These are incomplete!
 
 from typing import (
-    Any, Dict, Generic, TypeVar, Iterable, Tuple, Callable, Mapping, overload, Iterator,
-    Sized, Optional, List, Set, Sequence, Union, Reversible, MutableMapping, MutableSequence
+    Any, Container, Dict, Generic, TypeVar, Iterable, Tuple, Callable, Mapping, overload,
```

Comment:
This looks like it's just a cleanup. Can you roll that back? It's not pertinent to the purpose of the PR, and in general we don't accept cleanup diffs (they just cause churn without much value).